addons/spreadsheet/static/src/pivot/pivot_runtime.js

```diff
@@ -50,93 +55,75 @@ export class PivotRuntimeDefinition {
 }
 
 /**
- * Represent a measure in a pivot. A measure is a field that is aggregated.
+ * @param {Fields} fields
+ * @param {PivotMeasureDefinition} measure
+ * @returns {PivotMeasure}
  */
-export class PivotMeasure {
-    /**
-     * @param {Fields} fields All fields of the model
-     * @param {string} measureName Name of the measure
-     */
-    constructor(fields, measureName) {
-        this._fields = fields;
-        this._measureName = measureName;
-    }
-
-    /**
-     * Get the display name of the measure
-     * e.g. "__count" -> "Count", "amount_total" -> "Total Amount"
-     */
-    get displayName() {
-        return this._measureName === "__count"
-            ? _t("Count")
-            : this._fields[this._measureName].string;
-    }
-
-    /**
-     * Get the name of the measure, as it is stored in the pivot formula
-     */
-    get name() {
-        return this._measureName;
-    }
+function createMeasure(fields, measure) {
+    const name = measure.name;
+    const aggregator = measure.aggregator || fields[name]?.aggregator;
+    return {
+        nameWithAggregator: name + (aggregator ? `:${aggregator}` : ""),
+        /**
+         * Display name of the measure
+         * e.g. "__count" -> "Count", "amount_total" -> "Total Amount"
+         */
+        displayName: name === "__count" ? _t("Count") : fields[name].string,
```

Comment:
Can we add multiple times the same measure with a different aggregate ?

Reply:
We cannot for now

Reply:
Currently no, because it requires more changes:
- the formulas needs to change. Currently it's `=PIVOT.VALUE(1, "expected_revenue")` and it would need to become `=PIVOT.VALUE(1, "expected_revenue:sum")`
- the pivot model does not support it (I think)

But it's of course one of the next steps